src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -27,6 +28,7 @@
             <!-- Run after all custom normalizers -->
             <tag name="serializer.normalizer" priority="-1000" />
         </service>
+        <service id="Symfony\Component\Serializer\Normalizer\ObjectNormalizer" alias="serializer.normalizer.object" public="false" />
```

Comment:
It's probably a better idea to alias `Symfony\Component\Serializer\Normalizer\NormalizerInterface` and `Symfony\Component\Serializer\Normalizer\DenormalizerInterface` to the `serializer` service in case someone want to autowire a normalizer.